tests/integration/basic_c.cpp

```diff
@@ -74,6 +74,7 @@ class MemgraphConnection : public ::testing::Test {
     mg_session_params_set_port(params, memgraph_port);
     mg_session_params_set_sslmode(
         params, memgraph_ssl ? MG_SSLMODE_REQUIRE : MG_SSLMODE_DISABLE);
+    ASSERT_EQ(mg_connect(params, &session), 0);
```

Comment:
Makes sense, so the problem is the params->host pointing to an underlying const char * of a local std::string?

Reply:
That's the best explanation we have.